webviz_ert/data_loader/__init__.py

```diff
@@ -113,7 +113,7 @@ class DataLoader:
     _instances: MutableMapping[ServerIdentifier, "DataLoader"] = {}
 
     baseurl: str
-    token: str
+    token: Optional[str]
```

Comment:
Could you explain why this is optional now?

Reply:
Not really sure why pylint was not complaining before about this, it just started, I guess it was updated recently. 

The Dataloader `__new__` definition looked like this for quite some time 

```
def __new__(cls, baseurl: str, token: Optional[str] = None) -> "DataLoader":
```

So token is expected to he `Optional[str]`

Reply:
Thanks :)